hail_search/queries/sv.py

```diff
@@ -76,9 +77,20 @@ def _filter_annotated_table(self, ht, *args, parsed_intervals=None, exclude_inte
             if exclude_intervals:
                 interval_filter = ~interval_filter
             ht = ht.filter(interval_filter)
+        if padded_interval:
+            padding = int((padded_interval['end'] - padded_interval['start']) * padded_interval['padding'])
+            ht = ht.filter(hl.all([
+                ht.start_locus.contig == f"chr{padded_interval['chrom']}",
+                self._locus_in_range(ht.start_locus, padded_interval['start'], padding),
+                self._locus_in_range(ht.end_locus, padded_interval['end'], padding)
+            ]))
 
         return super()._filter_annotated_table(ht, *args, **kwargs)
 
+    @staticmethod
+    def _locus_in_range(locus, position, padding):
```

Comment:
Is [this](https://hail.is/docs/0.2/hail.expr.LocusExpression.html#hail.expr.LocusExpression.window) something that might be of use here?

Also, where does `3e8` come from?  Is it just "big number that's larger than the max pos on any config"?

Reply:
its copied from xpos_utils in seqr, its what we use in ES to validate. `window` seems better but I will add it is a subsequent PR to unblock this